README.md

```diff
@@ -69,6 +69,22 @@ Install Kerberos client to enable `skt` authentication to Beaker:
 Install `skt` dependencies following its
 [README.md](https://github.com/RH-FMK/skt).
 
+Run tests
+---------
+
+For running all tests write down:
+
+    python -m unittest discover tests
+
+For running some specific tests you can do this as following:
+
+    python -m unittest tests.test_patchwork
+
+Installation
```

Comment:
Can you also mention the `-e` option for developer modes, as skt has?

Reply:
Do you mean below `Developer Guide`?

Reply:
This is an old comment, you already resolved it with the developer guide

Reply:
If the line of the code that was commented on doesn't change, the comment stays on github, so you have to check the date and time when figuring out what comments are new

Reply:
Oh I see.